extra/mariabackup/xtrabackup.cc

```diff
@@ -1985,6 +1998,11 @@ xb_get_one_option(int optid,
     ADD_PRINT_PARAM_OPT(innobase_buffer_pool_filename);
     break;
 
+  case OPT_INNODB_FORCE_RECOVERY:
+
+    ADD_PRINT_PARAM_OPT(xtrabackup_innodb_force_recovery);
+    break;
```

Comment:
For backward compatibility, we would only want to display the parameter when it is set. Refer to the macro definition:
```c++
#define ADD_PRINT_PARAM_OPT(value)              \
  { \
    print_param_str << opt->name << "=" << value << "\n"; \
    param_set.insert(opt->name); \
  }
```
I think that this should be
```c++
  case OPT_INNODB_FORCE_RECOVERY:
    if (srv_force_recovery)
      ADD_PRINT_PARAM_OPT(srv_force_recovery);
    break;
```

Reply:
Oh, OK. That makes sense, will update in the next iteration.